bigtop.mk

```diff
@@ -294,7 +294,7 @@ IMPALA_RELNOTES_NAME=Impala
 IMPALA_PKG_NAME=impala
 IMPALA_BASE_VERSION=2.2.0
 IMPALA_PKG_VERSION=$(IMPALA_BASE_VERSION)+hdp2.3.0+rocana1.3.0
-IMPALA_RELEASE_VERSION=1
+IMPALA_RELEASE_VERSION=0.rc1
```

Comment:
http://serverfault.com/questions/454462/incremental-rpm-package-version-numbers-for-x-y-z-x-y-z-beta-or-alpha-rc